### ISSUE TITLE
Change default backlog size and make configurable

### ISSUE THREAD

**Issue by pietermees** [mafintosh/turbo-net#8]
Sending a lot of simultaneous requests causes the backlog to overflow and new connection attempts to be rejected. (see https://github.com/mafintosh/turbo-http/issues/12)

This is caused by the fixed default backlog size of `5` athttps://github.com/mafintosh/turbo-net/blob/master/src/turbo_net.c#L190.

Node's default is `511` (see https://nodejs.org/dist/latest-v9.x/docs/api/net.html#net_server_listen)
Node's `listen` also accepts a parameter to override the backlog size.

**Comment by mafintosh** [comment]
It's 511 now (you've fixed that :)) - we should still make it configurable

**Comment by pietermees** [comment]
Yes, and I guess we could change that TODO in the code from researching to making it configurable? Then users that don't like the default can do their own research ;)
